pcsx2/gui/GlobalCommands.cpp

```diff
@@ -1007,15 +1007,23 @@ void Pcsx2App::InitDefaultGlobalAccelerators()
 	GlobalAccels->Map(AAC(WXK_F4), "Framelimiter_MasterToggle");
 	GlobalAccels->Map(AAC(WXK_F4).Shift(), "Frameskip_Toggle");
 
-	/*GlobalAccels->Map( AAC( WXK_ESCAPE ),		"Sys_Suspend");
-	GlobalAccels->Map( AAC( WXK_F8 ),			"Sys_TakeSnapshot");
-	GlobalAccels->Map( AAC( WXK_F8 ).Shift(),	"Sys_TakeSnapshot");
-	GlobalAccels->Map( AAC( WXK_F8 ).Shift().Cmd(),"Sys_TakeSnapshot");
-	GlobalAccels->Map( AAC( WXK_F9 ),			"Sys_RenderswitchToggle");
+	// Doesn't read from the ini file at this point because `AppConfig::GetUiKeysFilename` is blank at this point!
+	// Used for custom hotkeys in the GUI.
```

Comment:
This portion of this comment is now inaccurate now that we have a better understanding of what's going on.

The problem is that at this point in the application initializing, it does not know if it's in portable or installed mode, and so it assumed installed mode, which i believe is the default.  https://github.com/PCSX2/pcsx2/blob/678829a5b2b8ca7a3e42d8edc9ab201bf00b0fe9/pcsx2/gui/AppInit.cpp#L479

This looks like it's been a long-standing issue, but not noticed because the majority of the keybindings are _actually_ set and used in `FrameForGS.cpp` by which point its figured things out.  This is something that should be addressed in the upcoming config work.  As a result, there may be some inconsistencies in the UI until that is fixed for users that use custom keybindings in portable mode.

I'd suggest changing this to something like:
```
// At this early stage of startup, the application assumes installed mode, so portable mode custom keybindings may present issues.
// Relevant - https://github.com/PCSX2/pcsx2/blob/678829a5b2b8ca7a3e42d8edc9ab201bf00b0fe9/pcsx2/gui/AppInit.cpp#L479
```